chalicelib/checks/wfr_checks.py

```diff
@@ -216,6 +251,7 @@ def md5run_start(connection, **kwargs):
     targets = []
     if kwargs.get('start_missing'):
         targets.extend(md5run_check_result.get('files_without_md5run', []))
+        targets.extend(md5run_check_result.get('released_without_md5run', []))
```

Comment:
I'm sure you already considered this but can you imagine any situation where we would want to be able to conditionally deal with these cases separately?  Probably not but just thought I'd throw it out there.